src/odemis/gui/cont/stream_bar.py

```diff
@@ -1198,6 +1200,51 @@ class SecomStreamsController(StreamBarController):
     Controls the streams for the SECOM and DELPHI live view
```

Comment:
Did you check that this works fine also on the DELPHI and SECOM GUI?

Reply:
Needs to be checked, not checked. Should I add the microscope files for DELPHI and SECOM in unit tests?

Reply:
No, I don't think the unit tests need to be extended. Just manually check that nothing weird happens. Either it's like the feature doesn't exist, or it shows the streams from the history (and also the other types of streams of the SECOM like SEM....)